Gulpfile.js

```diff
@@ -107,6 +112,7 @@ const projects = [
         cwd: "packages/table/",
         dependencies: ["core"],
         copy: false,
+        isotest: true,
```

Comment:
seems like isotest and karma are 1:1 -- if a package has karma tests then it'll also need isotests. so we probably don't need both tags. could rename them all to simply "test" ?